packages/opentelemetry/lib/plugin.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-case-declarations */
+
+import BasePlugin from '@appium/base-plugin';
+import { tracerProviderInstance } from './tracing/tracerProvider';
+
+const constants = {
+  ACTIVE: 'active',
+  INACTIVE: 'inactive',
+  SUCCESS: 'success',
+  FAILURE: 'failure'
+};
+
+export default class OpentelemetryPlugin extends BasePlugin {
+  constructor (pluginName, opts = {}) {
+    super(pluginName, opts);
+  }
+
+  static async updateServer (expressApp/*, httpServer*/) { // eslint-disable-line require-await
+    expressApp.post('/opentelemetry/config', OpentelemetryPlugin.setOpentelemetryConfig);
+    expressApp.get('/opentelemetry/config', OpentelemetryPlugin.getOpentelemetryConfig);
+    expressApp.get('/opentelemetry/status', OpentelemetryPlugin.getStatus);
+  }
+
+  static getOpentelemetryConfig (_req, res) {
+    res.send(JSON.stringify(tracerProviderInstance.getCurrentConfig()));
+  }
+
+  static setOpentelemetryConfig (_req, res) {
+    const opentelemetryBlob = _req.body;
+    try {
+      const exporterBlob = opentelemetryBlob.exporter;
```

Comment:
if body is not an object then not having this field should be an expected error (e.g. there should be an if condition to check it and throw an appropriate error)